opentelemetry-api/src/opentelemetry/_logs/__init__.py

```diff
@@ -39,6 +39,8 @@
     LogRecord,
     NoOpLogger,
     NoOpLoggerProvider,
+    ProxyLogger,
+    ProxyLoggerProvider,
```

Comment:
Is there a reason to add proxy classes here? They mainly exist to workaround an issue; we certainly don't want to expose to endusers.

Reply:
Only reason was to follow the implementation in trace, which defines the analogous classes in __init__. Let me know if you'd prefer to maintain that consistency or remove these imports.

My opinion is that I agree with your implication here and would remove them. The only disadvantage I can see to removing them from here is that I think pylint would complain if anyone imports them from _internal to subclass them in any bespoke implementations. Personally I think that's a good thing as it will force people to explicitly disable the lint warning on it and demonstrate that they know what they're doing in those environments.

Reply:
The trace package making the proxy classes public symbols was a mistake. They are purely internal implementation details to work around some issues. We should not add them here.

Reply:
Agreed, and should be reflected now in the latest commit